media/transcoder/transcoder_test.go

```diff
@@ -57,9 +58,12 @@ func TestJobTemplatesAndJobs(t *testing.T) {
 
 	bucketName := tc.ProjectID + "-golang-samples-transcoder-test"
 	inputURI := "gs://" + bucketName + "/" + testVideoFileName
+	inputOverlayImageURI := "gs://" + bucketName + "/" + testOverlayImageFileName
```

Comment:
This could be adjusted to `gs://cloud-samples-data/media/overlay.jpg` (possibly with a const or two).

Reply:
Sounds good. I updated the test file and deleted the test data.